code/modules/shuttle/emergency.dm

```diff
@@ -161,6 +166,21 @@
 /obj/machinery/computer/emergency_shuttle/proc/clear_recent_action(mob/user)
 	acted_recently -= user
 
+/obj/machinery/computer/emergency_shuttle/proc/pick_emag_amount()
```

Comment:
This doesn't need to be a separate proc, the code here can be put into the main proc it's used in.